resources/views/templates/admin-user/create.blade.php

```diff
@@ -0,0 +1,46 @@
+{{'@'}}extends('brackets/admin-ui::admin.layout.default')
+
+{{'@'}}section('title', trans('admin.{{ $modelLangFormat }}.actions.create'))
+
+{{'@'}}section('body')
+
+    <div class="container-xl">
+
+        <div class="card">
+
+            <{{ $modelJSName }}-form
+                :action="'{{'{{'}} url('admin/{{ $resource }}') }}'"
+                :activation="!!'@{{ $activation }}'"
+                @if($hasTranslatable):locales="@{{ json_encode($locales) }}"
+                :send-empty-locales="false"@endif
+
+                inline-template>
+
+                <form class="form-horizontal form-create" method="post" {{'@'}}submit.prevent="onSubmit" :action="this.action">
+
+                    <div class="card-header">
+                        <i class="fa fa-plus"></i> {{'{{'}} trans('admin.{{ $modelLangFormat }}.actions.create') }}
+                    </div>
+
+                    <div class="card-block">
```

Comment:
tu treba classu card-body namiesto card-block
```suggestion
                    <div class="card-body">
```